ksql-parser/src/test/java/io/confluent/ksql/parser/json/ExpressionSerializationTest.java

```diff
@@ -0,0 +1,62 @@
+package io.confluent.ksql.parser.json;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.confluent.ksql.execution.expression.tree.ArithmeticBinaryExpression;
+import io.confluent.ksql.execution.expression.tree.ColumnReferenceExp;
+import io.confluent.ksql.execution.expression.tree.DereferenceExpression;
+import io.confluent.ksql.execution.expression.tree.Expression;
+import io.confluent.ksql.execution.expression.tree.IntegerLiteral;
+import io.confluent.ksql.name.ColumnName;
+import io.confluent.ksql.name.SourceName;
+import io.confluent.ksql.schema.Operator;
+import io.confluent.ksql.schema.ksql.ColumnRef;
+import java.io.IOException;
+import java.util.Optional;
+import org.junit.Test;
+
+public class ExpressionSerializationTest {
```

Comment:
It's probably worth splitting this test into two: one for serialization and one for deserialization.  Most, (all?), the tests test _either_ serialization or deserialization, not both.  So splitting would be easy.

That way, people can find the unit tests for the sources files using IDE's inbuilt functionality to switch between code and test.

Same for the other combined tests.